lib/charts/chart.rb

```diff
@@ -17,20 +17,34 @@ def validate_arguments(data, options)
     raise ArgumentError.new('Data missing') if data.empty?
     raise ArgumentError.new('Data not an array') unless data.is_a? Array
     raise ArgumentError.new('Options missing') unless options.is_a? Hash
-    if options[:outer_margin] and !options[:outer_margin].is_a?(Numeric)
-      raise ArgumentError.new('outer_margin not a number')
+    if options[:outer_margin] && !options[:outer_margin].is_a?(Numeric)
+      raise ArgumentError, 'outer_margin not a number'
     end
-    validate_array_and_count(data, options, :colors)
-    validate_array_and_count(data, options, :labels)
+    is_an_array?(options, [:labels, :colors])
```

Comment:
`array?` rather than `is_an_array?` since the question mark asks that.  However, `validate_array_and_count` is likely better, depending on what the actual implementation details are... it tells me it is doing two things, where "is an array" only tells me that it is checking the type.